src/JWT/Serializers/Converters/DictionaryStringObjectJsonConverter.cs

```diff
@@ -117,7 +119,15 @@ private static void HandleValue(Utf8JsonWriter writer, string key, object object
                 }
                 default:
                 {
-                    writer.WriteNullValue();
+                    var payloadPartAsDictionary = objectValue.GetType()
```

Comment:
Apparently the code is duplicated. Let's create an internal helper method.

Or should we have this code here in the first place? Writing `null` seems to be a more expected behavior. Until we provide an option to control it

Reply:
Yeah, that sounds like a more safe approach

Reply:
Added a test to describe what I want to achieve. I want to be able to add a payload like this:
```
{
  "ExtraClaim": {
    "NestedProperty1": "Foo",
    "NestedProperty2": 3
  },
  "FirstName": "Jesus",
  "Age": 33
} 
```

Reply:
And the only way to achieve it is to have this code in the `default` case? It worries me, to be frank, as some sort of over-engineering.

Can you explain why do you need a complex object as a claim? Usually claims are a list of key-value pairs which are primitives or just strings.

Reply:
Sounds reasonable and worth giving it a try